app/controllers/admin/partners_controller.rb

```diff
@@ -24,15 +24,21 @@ def new
     def create
       # Find any existing publishers so we don't create duplicate entries
       @partner = partner
+      @organization = organization
 
       if @partner.persisted? && (@partner.partner? || @partner.admin?)
         flash.now[:alert] = "Email is already a partner"
         render :new
+      elsif @organization.persisted?
+        flash.now[:alert] = "The organization '#{params[:organization_name]}' already exists. Please have a partner of the organization add the user you want or ask Engineering team for assistance"
```

Comment:
I think i'd prefer getting the name from the object we've created
```suggestion
        flash.now[:alert] = "The organization '#{@organization.name}' already exists. Please have a partner of the organization add the user you want or ask Engineering team for assistance"
```